lib/file-stream.js

```diff
@@ -99,12 +99,15 @@ FileStream.prototype.pipe = function (dst) {
   var self = this
   var pipe = stream.Readable.prototype.pipe
 
-  if (dst && dst.nodeName === 'VIDEO') { // <video> tag
+  if (dst && dst.nodeName === 'VIDEO' || dst.nodeName === 'AUDIO') { // <video> and <audio> tag
```

Comment:
The if statement should be `dst && (dst.nodeName === 'VIDEO' || dst.nodeName === 'AUDIO')` because of order of operations and `dst` could be undefined. I'll fix.